targets/XPlatCppSdk/source/cppsdk/source/playfab/PlayFabSettings.cpp.ejs

```diff
@@ -48,7 +49,16 @@ namespace PlayFab
         fullUrl.reserve(1000);
 
         fullUrl += "https://";
-        fullUrl += titleId;
+
+        if(PlayFabSettings::verticalName != "")
```

Comment:
Doing a string comparison is way more expensive than doing a length check.
if(PlayFabSettings::verticalName.length > 0)